txp/Pos/Txp/Core/Tx.hs

```diff
@@ -66,7 +70,7 @@ topsortTxs toTx input =
             tsVisited %= HS.insert txHash
             let visitedNew = HS.insert txHash visitedThis
                 dependsUnfiltered =
-                    nub $ mapMaybe (\x -> HM.lookup (txInHash x) txHashes)
+                    nub $ mapMaybe txByInput
```

Comment:
@volhovM I am sure everything is fine here, but @volhovM posmoti zbl, a ne :loh: stav' v slack